tests/python_package_test/test_dask.py

```diff
@@ -200,8 +341,46 @@ def test_regressor_local_predict(client, listen_port):
     s2 = dask_regressor.to_local().score(X, y)
 
     # Predictions and scores should be the same
-    assert_eq(p1, p2)
     assert_eq(s1, s2)
+    assert_eq(p1, p2)
+
+
+@pytest.mark.parametrize('output', ['array', 'dataframe'])
+@pytest.mark.parametrize('group', [None, group_sizes])
+def test_ranker(output, client, listen_port, group):
+    X, y, w, g, dX, dy, dw, dg = _create_ranking_data(output=output, group=group)
+
+    dask_ranker = dlgbm.DaskLGBMRanker(time_out=5, local_listen_port=listen_port, seed=42, min_child_samples=1)
```

Comment:
I just made some changes in https://github.com/microsoft/LightGBM/pull/3786 to speed up the tests in this module. If it'll work for the tests you're adding in this PR, can you please make every `DaskLGBMRanker` and `LGBMRanker` use `n_iterations=10, num_leaves=10`? I found that training smaller models was able to make the tests faster without sacrificing any coverage.